pkg/statistics/analyze.go

```diff
@@ -83,6 +84,7 @@ func (a *AnalyzeResult) DestroyAndPutToPool() {
 type AnalyzeResults struct {
 	Err      error
 	ExtStats *ExtendedStatsColl
+	// The analyze results of each column/index.
 	Job      *AnalyzeJob
 	Ars      []*AnalyzeResult
```

Comment:
Could you please explain why this is considered an array, as we discussed earlier?

Reply:
Changed